### PR TITLE
Changes to create aggregated test to the data_module level instead of repo level

### PR DESCRIPTION
**Pull request by aagranados**

### CODE REVIEW

File: lib/domgen/ejb/generator.rb [hunk 1]
```diff
@@ -86,11 +86,13 @@
                                 'cdi_types_test.java.erb',
                                 type + '/java/#{repository.ejb.qualified_cdi_types_test_name.gsub(".","/")}.java')
     end
-    g.template_set(:"ejb_#{type}_qa_aggregate") do |template_set|
```
Comment (realityforge) [inline lib/domgen/ejb/generator.rb:89]: I am not sure why you changed this line but I would change it back unless you want to traul through our projects to see if it is needed ;-)

File: lib/domgen/ejb/model.rb [hunk 2]
```diff
@@ -141,7 +140,9 @@ def test_modules_map
     end
 
     facet.enhance(DataModule) do
+      include Domgen::Java::BaseJavaGenerator
       include Domgen::Java::EEClientServerJavaPackage
```
Comment (realityforge) [inline lib/domgen/ejb/model.rb:144]: New line after this!

File: lib/domgen/ejb/templates/aggregate_service_test.java.erb [hunk 3]
```diff
@@ -1,16 +1,15 @@
 /* DO NOT EDIT: File is auto-generated */
-package <%= to_package(repository.ejb.qualified_aggregate_service_test_name) %>;
+package <%= data_module.ejb.server_service_package %>;
```
Comment (realityforge) [inline lib/domgen/ejb/templates/aggregate_service_test.java.erb:2]: I would use the same pattern as used everywhere else and use `to_package`. This allows us to override the names of classes and packages of files. Very occasionally we need to do this for one reason or another which is why we use `to_pacage` everywhere else

File: lib/domgen/ejb/generator.rb (hunk 1 above)
Comment (aagranados) [reply lib/domgen/ejb/generator.rb:89]: Restored the deleted code.

**Review by realityforge — APPROVED** [review]

**Pull request merged**